include/gul14/ThreadPool.h

```diff
@@ -296,6 +296,9 @@ class ThreadPool : public std::enable_shared_from_this<ThreadPool>
      * // A task with a name
      * pool->add_task([]() { std::cout << "Task 4\n"; }, "Task 4");
      * \endcode
+     *
+     * \since GUL version 2.12.1, add_task() unconditionally accepts mutable function
```

Comment:
Why 'unconditionally', did it accept mutable function objects conditionally before?  
Just, because this can also be a typo because of re-formulation mid sentence.  
(_unconditionally accepts function objects_ vs _accepts mutable FO_)

Reply:
Well, there are overloads for `Function()` and `Function(ThreadPool&)`, and only the former was broken. I did not find a better concise formulation for that. :shrug: